dts/bindings/kscan/nuvoton,npcx-kscan.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX keyboard scan controller
+
+compatible: "nuvoton,npcx-kscan"
+
+include: [kscan.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+        required: true
+
+    clocks:
+        required: true
+
+    pinctrl-0:
+        required: true
+    pinctrl-names:
+        required: true
+
+    wui-maps:
```

Comment:
How about renaming it `ksi-wui`? Please refer to [nuvoton,npcx-host-sub.yaml](https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/espi/nuvoton%2Cnpcx-host-sub.yaml#L19). `wui-maps` is used for generic IO only. Adding a prefix indicating what kind of IO it is might be better.

Reply:
We named it to wui-maps because we can leverage the MACRO NPCX_DT_WUI_ITEMS_LIST to expand it and store the wui information to the array of the struct npcx_wui. Lets modify the yaml description to make it clearer to understand that this is for KSI's wui.

Reply:
Got it. Since the updated description has mentioned KSI pads, I'm okay with that.